capstone_project/src/main/java/com/google/sps/data/Resource.java

```diff
@@ -22,16 +22,14 @@
 
 /** Associates a category, from the Natural Language API taxonomy, to a link to a resource */
 public class Resource {
-  private final DatastoreService datastore;
+  private DatastoreService datastore;
```

Comment:
why is this no longer final?

Reply:
This was changed in error, I have made all fields in PostAnalysis and Resource final.